README.md

```diff
@@ -69,59 +72,73 @@
 
 ```bash
-pip3 install -r requirements.txt
-pip3 install -r requirements-pip.txt
+conda create --name pyani_env python=3.8 -y
+conda activate pyani_env
+conda install pyani
 ```
 
-### `bioconda`
+### `pip`
 
-With a working `anaconda` installation, install the `bioconda` and `conda-forge` channels:
+`pip` will install `pyani` and its Python dependencies, but not the third-party tools.
 
 ```bash
-conda config --add channels defaults
-conda config --add channels bioconda
-conda config --add channels conda-forge
+pip3 install pyani
 ```
 
-Then install `pyani`:
+### Third-party tools
+
+Three alignment packages are required, to use all of `pyani`'s methods: `mummer`, `BLAST+`, and legacy `BLAST`.
+
+The simplest route to obtaining these tools is to use `conda`/`bioconda`:
 
 ```bash
-conda install pyani
+conda install mummer blast legacy-blast -y
 ```
 
-**NOTE:** The current `conda` version is *behind* the development version and has a different command-line interface. Documentation for the `conda` version (0.2.x) can be found at the link below:
-
-- [https://github.com/widdowquinn/pyani/blob/master/README_v_0_2_x.md](https://github.com/widdowquinn/pyani/blob/master/README_v_0_2_x.md)
+But they can also be installed by following instructions from the tools' own websites.
 
+- **BLAST+** (for `anib`) [ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/](ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/)
+- **legacy BLAST** (for `aniblastall`) [ftp://ftp.ncbi.nlm.nih.gov/blast/executables/release/LATEST/](ftp://ftp.ncbi.nlm.nih.gov/blast/executables/release/LATEST/)
+- **MUMmer** (for `anim`) [http://mummer.sourceforge.net/](ftp://ftp.ncbi.nlm.nih.gov/blast/executables/release/LATEST/)
 
-### Installing legacy BLAST
+### NOTE: Installing legacy BLAST
 
 The NCBI legacy `BLAST` package is deprecated and not recommended. However, if you wish to use `pyani blastall` or the `ANIblastall` method with the legacy `pyani` interface, you will require a locally-installed copy of this package. This is one of the packages linked in the `requirements-thirdparty.txt` file.
 
 ## Citing `pyani`
 
 A guide to citing `pyani` is included in the file [`CITATIONS`](CITATIONS)
 
-## Documentation
+## Documentation (v0.3)
 
 This `README.md` file provides a quick overview and walkthrough for **THE DEVELOPMENT VERSION (v0.3+) OF `pyani`**, and full documentation can be found at the link below:
 
 - [`pyani` v0.3+ Documentation](https://readthedocs.org/projects/pyani/)
 
-### Older documentation
+**THIS README AND THE DOCUMENTATION AT `READTHEDOCS` REFERS TO A DEVELOPMENT VERSION OF `PYANI` (v0.3+). IT HAS A DIFFERENT COMMAND-LINE INTERFACE THAN THE STABLE `PYANI` VERSION (v0.2.x).**
+
+**THE STABLE VERSION OF `PYANI` (v0.2) DOES NOT HAVE THE `pyani` COMMAND**
```

Comment:
What do you think of putting things like this inside of a callout box? I have not been able to find a quick solution for doing this in GitHub Markdown, but even if there is no way to do it in pure markdown, html/css will work. There are a few places where I think it would make it easier to differentiate **notate bene** from the rest of the text—perhaps more-so than bold all-caps does.

Reply:
In the appropriate circumstances, a callout box would be perfect.

As you note, it relies on the renderer, and its willingness to parse and present HTML/CSS. Here I'm targeting GitHub's default rendering, and the default Jekyll/GitHub Pages setup. They're rather limited in scope - as is Markdown.

You'll see on the ReadTheDocs pages I make liberal use of callouts, etc. when it's available.

Reply:
I've always found html and css to interact well with markdown, even without something like Jekyll; I'll see if I can get something non-complex working.

Reply:
I guess the other consideration is that `README.md` takes the place of `README.txt`. The document needs to be legible when it's not viewed through a renderer - e.g. going through the doc with `less`. On balance I think it's best to keep the `README.md` formatting relatively light, to avoid a level of HTML/CSS that obscures reading of the text.

In practice, this means things like avoiding inline CSS at the top of the page and, since GitHub doesn't let you use external CSS for `README`s, I've stuck to the inbuilt Markdown formatting.

- ref: [https://github.com/github/markup](https://github.com/github/markup)